src/common/components/CardFaces.js

```diff
@@ -1,25 +1,39 @@
-export const cardFaces = [{
-    name: 'a',
-    face: '😛'
+import {
+  shuffleArray,
+  doubleArray
+} from "common";
+
+const cardSet = [{
+    name: 'tongue',
+    face: '😛',
+    isFaceUp: false
   },
   {
-    name: 'b',
-    face: '🤓'
+    name: 'nerd',
+    face: '🤓',
+    isFaceUp: false
   },
   {
-    name: 'c',
-    face: '😚'
+    name: 'grimace',
+    face: '😬',
+    isFaceUp: false
   },
   {
-    name: 'd',
-    face: '😬'
-  },
-  {
-    name: 'e',
-    face: '😅'
-  },
-  {
-    name: 'f',
-    face: '😩'
+    name: 'agony',
+    face: '😩',
+    isFaceUp: false
   },
 ]
+
+export const cardFaces = () => shuffleArray(doubleArray(cardSet))
+
+// {
```

Comment:
To keep the code clean, I recommend removing commented out code when submitting pull requests.

Reply:
I was just keeping those in reserve in case I wanted to bump back up to 6 card faces later -- but I can totally type a couple emojis again :)

Reply:
That's fine. I do recommend removing commented code before merging in the pull request, though :)

If the PR is work in progress, you can add `WIP` to the title in GitHub.